index.js

```diff
@@ -142,7 +214,19 @@ function getAllMoviesReleasedAtOrBeforeYear() {}
  *  getBiggestBoxOfficeMovie(movies);
  *  //> "Incredibles 2"
  */
-function getBiggestBoxOfficeMovie() {}
+function getBiggestBoxOfficeMovie(movies) {
+  let mostMoney = movies[0];
+  
+  if (movies.length === 0) {
+    return null;
+  } 
+  for (let i = 1; i < movies.length; i++) {
+    if (Number(movies[i].boxOffice.substr(1).replace(/,/g, "")) > Number(mostMoney.boxOffice.substr(1).replace(/,/g, ""))) {
```

Comment:
Great Regex!  Notice how we are calling it twice?  This is a perfect place for a helper function!  
with something like `replaceCommas()` we could change our code to
`if(replaceCommas(movies[i]) > replaceCommas(mostMoney)) {  <do code here>}`. See if you can make it happen!